lib/modules/datasource/cdnjs/index.ts

```diff
@@ -43,33 +43,24 @@ export class CdnJsDatasource extends Datasource {
     const result = Result.parse(config, ReleasesConfig)
       .transform(({ packageName, registryUrl }) => {
         const [library] = packageName.split('/');
-        const assetName = packageName.replace(`${library}/`, '');
 
-        const url = `${registryUrl}libraries/${library}?fields=homepage,repository,assets`;
+        const url = `${registryUrl}libraries/${library}?fields=homepage,repository,versions`;
 
         const schema = z.object({
           homepage: Homepage,
           repository: Repository,
-          assets: Assets.transform((assets) =>
-            assets
-              .filter(({ files }) => files.includes(assetName))
-              .map(({ version, sri }) => {
-                const res: Release = { version };
-
-                const newDigest = sri?.[assetName];
-                if (newDigest) {
-                  res.newDigest = newDigest;
-                }
-
-                return res;
-              }),
+          versions: Versions.transform((versions) =>
+            versions.map((version) => {
+              const res: Release = { version };
+              return res;
+            }),
```

Comment:
Previously, we needed to create the schema instance dynamically, because we needed transform to be closed over `assetName` variable. Now it seems like the schema could be defined statically in single place.